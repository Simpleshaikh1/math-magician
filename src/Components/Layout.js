// eslint-disable-next-line import/no-extraneous-dependencies
import { Outlet } from 'react-router-dom';
import Header from './Header';

const Layout = () => (
  <>
    <Header />
    <Outlet />
  </>
);
export default Layout;
