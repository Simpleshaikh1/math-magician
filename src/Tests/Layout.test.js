import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import Layout from '../Components/Layout';

test('renders Layout correctly', () => {
  const { container } = render(
    <MemoryRouter>
      <Layout />
    </MemoryRouter>,
  );
  expect(container).toMatchSnapshot();
});