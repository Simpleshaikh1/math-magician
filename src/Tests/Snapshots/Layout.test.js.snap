// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Layout correctly 1`] = `
<div>
  <header>
    <h1>
      Math Magicians
    </h1>
    <nav>
      <ul>
        <li>
          <a
            href="/"
            style="color: orange;"
          >
            Home
          </a>
        </li>
        <li>
          <a
            href="/calculator"
          >
            Calculator
          </a>
        </li>
        <li>
          <a
            href="/quotes"
          >
            Quotes
          </a>
        </li>
      </ul>
    </nav>
  </header>
</div>
`;